tools/license2rtf.mjs

```diff
@@ -287,19 +286,18 @@ class RtfGenerator extends Stream {
   }
 }
 
-
-const stdin = process.stdin;
-const stdout = process.stdout;
-const lineSplitter = new LineSplitter();
-const paragraphParser = new ParagraphParser();
-const unwrapper = new Unwrapper();
-const rtfGenerator = new RtfGenerator();
-
 stdin.setEncoding('utf-8');
 stdin.resume();
 
-stdin.pipe(lineSplitter);
-lineSplitter.pipe(paragraphParser);
-paragraphParser.pipe(unwrapper);
-unwrapper.pipe(rtfGenerator);
-rtfGenerator.pipe(stdout);
+async function main() {
+  await pipeline(
+    stdin,
+    new LineSplitter(),
+    new ParagraphParser(),
+    new Unwrapper(),
+    new RtfGenerator(),
+    stdout,
+  );
+}
+
+main().catch(console.error);
```

Comment:
nit: alternatively, we could also remove the `await` since there is nothing afterwards that depends on the completion of the pipeline call
```suggestion
pipeline(
  stdin,
  new LineSplitter(),
  new ParagraphParser(),
  new Unwrapper(),
  new RtfGenerator(),
  stdout,
);
```

Reply:
fixed, PTAL